pkg/monitor/querier.go

```diff
@@ -151,6 +152,16 @@ func (monitor *agentMonitor) GetAgentConditions(ovsConnected bool) []v1beta1.Age
 	}
 }
 
+// GetAgentInfo gets the antrea agent information.
+func (monitor *agentMonitor) GetAgentInfo() *v1beta1.AntreaAgentInfo {
+	info, err := monitor.client.ClusterinformationV1beta1().AntreaAgentInfos().Get(monitor.GetSelfNode().Name, metav1.GetOptions{})
```

Comment:
As I have mentioned previously, you cannot get the latest information for CLI if you query CRD directly.
In my point of view, CLI should get the latest information rather than the information which may be a minute late.
If so, you can extract the AntreaAgentInfo construction part of code here from monitor.go.
@jianjuns @tnqn What do you think?

Reply:
I would second what @mengdie-song said.

Reply:
Thank you Mengdie and Jianjun! I have updated the code.